src/guiapi/src/display_helper.c

```diff
@@ -34,9 +36,15 @@ void render_text_align(rect_ui16_t rc, const char *text, font_t *font, color_t c
         display->fill_rect(rc_l, clr0);
         display->fill_rect(rc_r, clr0);
     } else {
-        point_ui16_t wh_txt = font_meas_text(font, text);
+        char *str_tmp = (char *)0x10000000; // ~ PNG-buffer (CCM RAM)
+        strcpy(str_tmp, text);
+        if (ml_mode == ML_MODE_EXT) {
+            set_instance(((window_text_t *)context)->pml_mode);
+            str2multiline(str_tmp, ((window_text_t *)context)->line_width);
```

Comment:
you should be able to calculate `line_width ` from `font` and `rc`
try pass just `pml_mode` argument instead `context`